tools/client/Cargo.toml

```diff
@@ -21,7 +21,7 @@ base64.workspace = true
 clap = { version = "4.0.29", features = ["derive"] }
 env_logger.workspace = true
 jwt-simple = "0.11.4"
-kbs_protocol = { git = "https://github.com/confidential-containers/guest-components" }
+kbs_protocol = { git = "https://github.com/chendave/guest-components.git", branch = "fvp" }
```

Comment:
Do we still need this?

Reply:
@fitzthum, we don't, just need that pending PR to be merged, then this I will be updated. :)